crates/tx-parser/src/filter_types.rs

```diff
@@ -0,0 +1,61 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use strata_primitives::{
+    buf::Buf32,
+    l1::BitcoinAddress,
+    params::{DepositTxParams, RollupParams},
+    sorted_vec::SortedVec,
+};
+
+use crate::utils::{generate_taproot_address, get_operator_wallet_pks};
+
+#[derive(Clone, Debug, BorshSerialize, BorshDeserialize)]
+pub struct TxFilterConfig {
+    /// For checkpoint update inscriptions.
+    pub rollup_name: String,
+
+    /// For addresses that we expect spends to.
+    pub expected_addrs: SortedVec<BitcoinAddress>,
+
+    /// For blobs we expect to be written.
+    pub expected_blobs: SortedVec<Buf32>,
+
+    /// For deposits that might be spent from.
+    pub expected_outpoints: SortedVec<Outpoint>,
+
+    /// Deposit config that defines the structure we expect in the utxo
```

Comment:
```suggestion
    /// Deposit config that defines the structure we expect in the utxo.
```
The only docstring in this struct that does not have a `.` in the end. 😓
Also I don't like "we" in docstrings?
Who are "we"? Alpen Labs, Strata, Users, Developers, Sequencer, Bridge Operator, Humans?

Reply:
I always use it as the instructive we that an instructor would use when working through a problem on a whiteboard.  Related to but different from the royal we.